src/wp-admin/includes/translation-install.php

```diff
@@ -35,23 +34,26 @@ function translations_api( $type, $args = null ) {
 	$res = apply_filters( 'translations_api', false, $type, $args );
 
 	if ( false === $res ) {
-		$url = $http_url = 'https://api.wordpress.org/translations/' . $type . '/1.0/';
-
+		$stats = array(
+			'wp_version' => '4.9.9', //since CP is based on 4.9.9
+			'locale'     => get_locale(),
+			'version'    => $args['version'], // Version of plugin, theme or core
+		);
 		$options = array(
 			'timeout' => 3,
-			'body' => array(
-				'wp_version' => $wp_version,
-				'locale'     => get_locale(),
-				'version'    => $args['version'], // Version of plugin, theme or core
-			),
 		);
-
-		if ( 'core' !== $type ) {
+		if( 'core' !== $type ) {
+			$stats['cp_version'] = $cp_version;
+			$url = 'https://api.wordpress.org/translations/' . $type . '/1.0/';
 			$options['body']['slug'] = $args['slug']; // Plugin or theme slug
+			$request = wp_remote_post( $url, array_merge( $options, array(
+				'body' => $stats
+			) ) );
+		} else {
+			$url = add_query_arg($stats, 'https://translate.classicpress.net/wp-content/translations/' . $type . '/1.0.0/translations.json');
```

Comment:
my only dount is that 1.0.0 hardcoded but I think that we can change it after beta 2 to something different, so is not a blocker.

Reply:
Yes, this will need to be updated later, and we'll also need to make the translation server understand how to generate different versions.  I think it's ok for now though.